src/test/java/com/epam/tc/hw3/testcases/BaseTest.java

```diff
@@ -0,0 +1,41 @@
+package com.epam.tc.hw3.testcases;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeSuite;
+
+
+public class BaseTest {
+    static WebDriver testDriver;
+
+    public static String getBaseURL() {
+        return baseURL;
+    }
+
+    private static final String baseURL = "https://jdi-testing.github.io/jdi-light/index.html";
```

Comment:
Why static? Do you want only one driver at the moment?

Reply:
More convenient to access, no parallel tests (yet). Fixed.